src/pocketmine/Player.php

```diff
@@ -2980,6 +2980,14 @@ public function sendTip($message){
 		$pk->message = $message;
 		$this->dataPacket($pk);
 	}
+	
+	public function whisper($from, $message){
+		$pk = new TextPacket();
+		$pk->type = TextPacket::TYPE_WHISPER;
+		$pk->source = $from instanceof Player ? $from->getDisplayName() : $from;
```

Comment:
I took sendMessage as example, where the parameter could be a string or a TextContainer. If this causes misunderstandings I'll limit it to a string.